### ISSUE TITLE
Disambiguation error

### ISSUE THREAD

**Issue by mk-uc** [jgm/citeproc#133]
I know this has been addressed before, [here](https://github.com/jgm/citeproc/issues/116), [here](https://github.com/jgm/citeproc/issues/90), and [here](https://github.com/jgm/citeproc/issues/44) but I'm still running into a similar problem. 

I cite a book and then in subsequent citations, it includes the year for seemingly no reason. Any help would be much appreciated!

Example bibtex:

```
@Book{Memmi2009,
author = {Memmi, Albert}, 
title = {Testament insolent}, 
publisher = {Odile Jacob}, 
address = {Paris}, 
year = {2009}, 
langid = {French}, 
keywords = {Memmi},}
```

Produces (note the inclusion of the year):
![Screenshot 2023-05-03 at 11 24 19 AM](https://user-images.githubusercontent.com/7539590/235979024-eda62542-e4e2-4571-a9fa-2ce3f0b76edb.png)

I am using the latest Chicago Manual of Style 17th edition (full note, short title subsequent) link [here](https://www.zotero.org/styles/chicago-fullnote-bibliography-short-title-subsequent)

**Comment by jgm** [comment]
It would be good to have a reproducible test case.  Minimal bib file and md file for reproducing the issue.

**Comment by jgm** [comment]
Yes, disambiguation occurs when you have another reference with the same author, title, etc.

I would try trimming down your bibtex file little by little til you get a minimal case that can reproduce the issue.

**Comment by thjbdvlt** [comment]
I face a similar issue (year added for no reason), that happens when cases like:

```markdown
line one [@vandendorpe1999, 1])
line two [@vandendorpe1999, 1])
line three [@poliak2006 12]
line four [@vandendorpe1999])
```

Line four is "VANDENDOROP, 1999, op. cit." but there have only one item of him, so no disambiguation is needed.
It seems to be caused by the the fact that line two is "ibid-with-locator".
Because if i do this, there is no issue:

```markdown
line one [@vandendorpe1999, 1])
line two...
line three [@poliak2006 12]
line four [@vandendorpe1999])
```
